### PR TITLE
New elections service

### PR DESCRIPTION
**Pull request by ben-willis**
Just the entities and the back end service. No integration with the existing site so nothing to break hopefully.

### CODE REVIEW

**Review by d-nagy — COMMENTED** [review]
Looks good to me Ben.

I ran it to check though, and found an error with the current election system, in that we use `mime.extension` instead of `mime.getExtension` in election.js. I don't know if you want to fix that in this PR or I can make a new one.

**Comment by ben-willis** [comment]
@d-nagy Might as well fix it in this PR. There will be another PR later integrating this election service in with everything else.

File: src/routes/admin/elections.js [hunk 1]
```diff
@@ -203,7 +203,7 @@ router.get('/:election_id/:position_id/delete', function (req, res, next) {
 router.post('/:election_id/:position_id/newnominee', upload.single('manifesto'),function (req, res, next) {
 	moveManifesto = new Promise(function(resolve, reject){
 		if (req.file) {
-			var manifesto_name = slug(req.body.name)+'-'+shortid.generate()+'.'+mime.extension(req.file.mimetype);
+			var manifesto_name = slug(req.body.name)+'-'+shortid.generate()+'.'+mime.getExtension(req.file.mimetype);
```
Comment (ben-willis) [inline src/routes/admin/elections.js:206]: @d-nagy  Look good?

Comment (d-nagy) [reply src/routes/admin/elections.js:206]: Yeah that was the only thing, cheers!

**Pull request merged**